src/ui/virtualkeyboard/virtualkeyboard.cpp

```diff
@@ -272,6 +272,13 @@ void VirtualKeyboard::updateInputPanel(InputContext *inputContext) {
     updateCandidate(inputContext);
 }
 
+void VirtualKeyboard::initVirtualKeyboardService() {
+    service_ = std::make_unique<VirtualKeyboardService>(this);
+    bus_->addObjectVTable("/virtualkeyboard",
+                          "org.fcitx.virtualkeyboard.service", *service_);
```

Comment:
We use interface name like [lower case reversedomain].Fcitx.CamelCaseServiceNameVersion 
So ideally should be called org.fcitx.Fcitx.VirtualKeyboard1

Reply:
ok